### PR TITLE
feat: add new heart and follow icons

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [ONYX-1286] [ONYX-1293] 

### Description

To address https://artsyproduct.atlassian.net/browse/ONYX-1286 and https://artsyproduct.atlassian.net/browse/ONYX-1293 we need to add new icons
image
image

[ONYX-1286]: https://artsyproduct.atlassian.net/browse/ONYX-1286?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ONYX-1293]: https://artsyproduct.atlassian.net/browse/ONYX-1293?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/svgs/NewFillHeartIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Icon, IconProps, Path } from "./Icon"
+import { useColor } from "../utils/hooks"
+
+export const NewFillHeartIcon = ({ fill, ...restProps }: IconProps) => {
```
Comment (dariakoko) [inline src/svgs/NewFillHeartIcon.tsx:4]: NewFillHeartIcon and NewHeartIcon are not the perfect names...
After [the feature](https://artsyproduct.atlassian.net/browse/ONYX-1286) is release and corresponding feature flag is no longer in use we can adjust the FillHeartIcon and HeartIcon components and remove those added in this PR

File: src/svgs/FollowArtistFillIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Icon, IconProps, Path } from "./Icon"
+import { useColor } from "../utils/hooks"
+
+export const FollowArtistFillIcon = ({ fill, ...restProps }: IconProps) => {
+  const color = useColor()
+
+  return (
+    <Icon {...restProps} viewBox="0 0 17 13">
```
Comment (gkartalis) [inline src/svgs/FollowArtistFillIcon.tsx:8]: 🤔 agree with @araujobarret point around the screenshots, if I remember correctly almost all of our icons have `viewBox="0 0 18 18"` screenshots would be awesome in order to see if these look right or not.

I think you don't need to add a story, I think it is already handled and maps over all of the icons

Comment (dariakoko) [reply src/svgs/FollowArtistFillIcon.tsx]: Addressed both comments, thanks!

**Review by gkartalis — APPROVED** [review]
thank you!

**Pull request merged**